.github/workflows/release.yml

```diff
@@ -4,6 +4,12 @@ on:
     branches:
       - "main"
   workflow_dispatch:
+    inputs:
+      stage:
+        description: Stage to deploy to
+        type: environment
+        required: true
+        default: dev
```

Comment:
While we are in alpha we can deploy to `prod` stage by default. For none main branch triggers we can use the snapshot release and deploy to dev.

After the 1.0.0 release we will change that logic.

What do you think?

Reply:
So should we remove the "stage" parameter and set it to "prod" if on main and "dev" otherwise?
I guess for now we probably only need one stage, right? So either prod or dev

Reply:
I would say `prod` for main branch and `dev` for others.

Reply:
Done :)